concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md

```diff
@@ -0,0 +1,91 @@
+To customize appearance of all Autocomplete suggestions, define an [itemTemplate](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate). In React, implement the [itemRender](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemRender) or [itemComponent](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemComponent) function. To customize an individual suggestion, specify the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property of the item. Note that Angular and Vue use [custom templates](/Documentation/Guide/UI_Components/Common/Templates/#Custom_Templates) instead of the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property. In React, specify the [render](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#render) or [component](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#component) property.
```

Comment:
Please do not use API members as ordinary words in a sentence. And yes, mind the articles.

```suggestion
To customize the appearance of all Autocomplete suggestions, define an [item template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate). In React, implement the [itemRender](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemRender) or [itemComponent](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemComponent) function. To customize an individual suggestion, specify the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property of the item. Note that Angular and Vue use [custom templates](/Documentation/Guide/UI_Components/Common/Templates/#Custom_Templates) instead of the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property. In React, specify the [render](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#render) or [component](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#component) property.
```